app/services/idv/session.rb

```diff
@@ -2,7 +2,7 @@ module Idv
   class Session
     VALID_SESSION_ATTRIBUTES = [
       :resolution, :vendor, :applicant, :params, :profile_id, :recovery_code,
-      :financials_confirmation, :phone_confirmation
+      :financials_confirmation, :phone_confirmation, :profile_confirmation
```

Comment:
not sure if rubocop would dislike this, but I find arrays like this easier to read when each item is on its own line. AND alphabetized, if possible.